frontend/src/common/types/carousel/carousel-responsive-prop.type.ts

```diff
@@ -0,0 +1,11 @@
+type CarouselResponsiveBreakpoints = {
+  [key: string]: {
+    breakpoint: {
+      max: number;
+      min: number;
+    };
+    items: number;
+  };
```

Comment:
[Record](https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type) is preferable way to construct object types 

upd. can we reuse ResponsiveType from react-multi-carousel?
```
export interface ResponsiveType {
    [key: string]: {
        breakpoint: {
            max: number;
            min: number;
        };
        items: number;
        partialVisibilityGutter?: number;
        paritialVisibilityGutter?: number;
        slidesToSlide?: number;
    };
}
```